packages/curve-ui-kit/src/themes/themes.ts

```diff
@@ -16,7 +16,7 @@ const generateTheme = (theme: ThemeKey, options: DesignOptions = {}): Theme => {
     key: theme,
     design: { ...design, options },
     palette: createPalette(paletteMode(theme, options), design),
-    typography,
+    typography: typography,
```

Comment:
Hah, not sure what the intention here was 😅

Reply:
ah lol I wanted to ctrl+click to the typography definition, forgot to remove this.